modules/pantheon/pantheon_apachesolr/pantheon_apachesolr.module

```diff
@@ -203,7 +211,7 @@ function pantheon_apachesolr_post_schema_exec($schema) {
     variable_set('pantheon_apachesolr_schema', $schema);
   }
 
-  return $response;
+  return $response || $status;
```

Comment:
Returning `$response || $status` is not desirable, because these variables are of different data types, which makes things hard on the caller.  Currently, there is only one caller of pantheon_apachesolr_post_schema_exec , and that is pantheon_apachesolr_post_schema_form_submit, which only cares whether the call worked or not.  I would therefore recommend that we change this to simply 'return $status', and change the documentation appropriately to indicate that the only result available is whether the call worked or not.  This wouldn't be a good thing to do if we expected some caller would need to access the $response body, but at the moment that does not appear to be necessary.